.github/workflows/maven-build.yml

```diff
@@ -90,6 +90,7 @@ jobs:
           echo "ARTIFACTID=$ARTIFACTID" >> $GITHUB_OUTPUT
 
       - name: Maven Build and Sonar
+        if: ${{ inputs.is_release == 'true' }}
```

Comment:
Can this be is_snapshot.  As we want PRs to run this, and they are technically a release

Reply:
grr you're right.